### PR TITLE
Episode 2 Draft

### PR DESCRIPTION
**Pull request by bstrie**
Still need the filename and the publishing date to be filled in.

### CODE REVIEW

**Comment by bstrie** [comment]
Good to be merged if you just want to give it a once-over check for obvious errors.

File: _episodes/001-ruma.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+date: 2019-08-08T11:45:00+0000
+file: https://audio.rustacean-station.org/file/rustacean-station/rustacean-station-e001-ruma-matrix-jimmy-cuadra.mp3
+duration: "1:03:29" #audio length in min
+length: "60946851" #filesize in byte
+
+# https://github.com/jekyll/jekyll/issues/7744
+layout: episode
+---
+
+We interview Jimmy Cuadra about Matrix, an open and decentralized communication protocol, and his implementation in Rust known as [Ruma](https://github.com/ruma/ruma).
```
Comment (killercup) [inline _episodes/001-ruma.md]:
```suggestion
We interview Jimmy Cuadra about [Matrix](https://matrix.org/), an open and decentralized communication protocol, and his implementation in Rust known as [Ruma](https://github.com/ruma/ruma).
```

File: _episodes/001-ruma.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+ - Twitter: [@rustaceanfm](https://twitter.com/rustaceanfm)
+ - Discord: [Rustacean Station](https://discord.gg/cHc3Gyc)
+ - Github: [@rustacean-station](https://github.com/rustacean-station/)
+ - Email: [hello@rustacean-station.org](mailto:hello@rustacean-station.org)
+
+### Timestamps & referenced resources
+
+* 0:01:35 - Meet Jimmy Cuadra
```
Comment (killercup) [inline _episodes/001-ruma.md]: At least pocket casts parses these time stamps as links when written like `[@01:35]`

Comment (bstrie) [reply _episodes/001-ruma.md]: Interesting, I'll try that.

**Pull request merged**